Add osu! Talk Event: Overcoming Obstacles newspost (#7351)
ppy/osu-wiki: news/2022-05-19-osu-talk-event-overcoming-obstacles.md

---
layout: post
title: "osu! Talk Event: Overcoming Obstacles"
date: 2022-05-19 08:00:00 +0000
---

Mark your calendars, set your alarms, and join us for the first ever osu! talk event, where you can make your voice heard.

![](/wiki/shared/news/2022-05-19-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)

> Unlike many events that involve guest speakers, this one is open for anyone to speak — all you need is a story to share. Each event centers around a certain theme. For this iteration, all talks will center around the theme of **overcoming obstacles**.

Reviewer: ```suggestion
Unlike many events that involve guest speakers, this one is open for anyone to speak — all you need is a story to share. Each event centers around a certain theme, and for this iteration, all talks will have the theme of **overcoming obstacles**.
```

Here's a rundown of how the event works:

- **The event will last 3 days total**, from May 20 - 22, 2022.
- **The speaker will use a Discord stage channel** in the [official osu!dev Discord server](https://discord.gg/ppy) to give their talk.
- **Talks will be less than 18 minutes long.** This is the maximum allowed length, not a goal to aim for — even a 3-minute talk can have massive impact.

> - The stories shared do not necessarily have to be about osu!, but because it is an osu!-related event, the speakers are recommended to try connecting their story or the lesson behind their story to osu! in some way.

Reviewer: ```suggestion
- The stories shared do not necessarily have to be about osu!, but because this is an osu!-related event, speakers are recommended to try connecting their story or the lesson behind it to osu! in some way.
```

| 13:00 UTC | [Axiqn](https://osu.ppy.sh/users/21130016) | "Focusing on weaknesses" |
| 13:30 UTC | [UserAlsoExists](https://osu.ppy.sh/users/19036931) | "The "play less" meta" |
| 14:30 UTC | [-Liability](https://osu.ppy.sh/users/12260184) | "Doubting yourself" |
| 15:00 UTC | [Astral52](https://osu.ppy.sh/users/11936432) | "Opening the mind to have fun: How to build a less toxic mentality" |

### Sunday, May 22, 2022

[**Sign up for this event on Discord to be notified when it starts!**](https://discord.gg/z62bbJHA?event=976725279126982656)

| Time (UTC) | Speaker | Title |
| :-: | :-- | :-- |
| 18:00 UTC | [cristi2708](https://osu.ppy.sh/users/7552300) | "A perspective on switches and tapping techinques" |
| 18:30 UTC | [PruceStrats](https://osu.ppy.sh/users/16518886) | "Overcoming adversity in daily life" |
| 19:00 UTC | [-TunaSliders-](https://osu.ppy.sh/users/15420104) | "Using ideas from osu! to view life differently" |
| 19:30 UTC | [WaifuMaterial](https://osu.ppy.sh/users/14592606) | "Overcoming social anxiety" |
| 20:00 UTC | [chiv](https://osu.ppy.sh/users/6701656) | "The problematic state of osu! tournaments" |

> The speakers were selected through a public speaker nomination form. If you believe that you or someone you know would be fit to give a talk in future events, please fill out the [speaker nomination form](https://forms.gle/HCD6ac8JwURGh8zx8). The same nomination form will be used for all future osu! talk events.

Reviewer: ```suggestion
```

> 

Reviewer: (reviewing on mobile with no multi-line selection)

```suggestion
```

---

> Join us this weekend for this inspirational event! Most importantly, be sure to spread the word to anyone you can. We know the folks in `#general` don't want to miss this.

Reviewer: ```suggestion
Join us this weekend for a hopefully inspirational event! Most importantly, be sure to spread the word to anyone you can. We know the folks in `#general` don't want to miss this.
```

Reviewer 2: also what is `#general` here?

Reviewer: it's probably a pun on "in general"

> If you are interested in being considered to speak in future events, please fill out the [speaker nomination form](https://forms.gle/HCD6ac8JwURGh8zx8).

Reviewer: just to remove the duplicate paragraph

```suggestion
The speakers were selected through a public speaker nomination form. If you believe that you or someone you know would be fit to give a talk in future events, please fill out the [speaker nomination form](https://forms.gle/HCD6ac8JwURGh8zx8). The same nomination form will be used for all future osu! talk events.
```
